man/tree_abstract.Rd

```diff
@@ -4,12 +4,10 @@
 \alias{tree_abstract}
 \title{Abstraction layer function}
 \usage{
-tree_abstract(model, inputdata)
+tree_abstract(model)
 }
 \arguments{
 \item{model}{The model to organise}
-
-\item{inputdata}{The data used to train the model}
 }
 \value{
 A structure with the class name "tree_model" which contains a list of all the relevant model data, 
```

Comment:
Need to delete the 'and the input data used to train the model.' part of the description as well